Dockerfile

```diff
@@ -9,7 +9,7 @@ ARG EFFECTIVE_VERSION
 RUN make install EFFECTIVE_VERSION=$EFFECTIVE_VERSION
 
 ############# base
-FROM eu.gcr.io/gardener-project/3rd/alpine:3.12.3 AS base
+FROM alpine:3.12.3 AS base
```

Comment:
Can you revert these changes again to use the GCR for the images? :)

Reply:
We cannot use GCR image for other ports as currently available images are build on AMD64, and therefore binaries are not compatible.
Shall I keep my changes in separete new Dockerfile?
or 
Shall I request to push images for PPC too? If so How can I request?

Reply:
Another alternative is to use an ARG e.g., `ARG BASE` where `eu.gcr.io/gardener-project/3rd/` is the default.

https://docs.docker.com/engine/reference/builder/#understand-how-arg-and-from-interact.

Reply:
Please review now.